tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -137,3 +137,13 @@ sudo umount $MOUNT_DIR
 gcsfuse --only-dir testDir --implicit-dirs $TEST_BUCKET_NAME $MOUNT_DIR
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/list_large_dir/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR --testbucket=$TEST_BUCKET_NAME/testDir
 sudo umount $MOUNT_DIR
+
+# Run integration tests for read_large_files directory with static mounting
+gcsfuse --implicit-dirs $TEST_BUCKET_NAME $MOUNT_DIR
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/read_large_files/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR --testbucket=$TEST_BUCKET_NAME
+sudo umount $MOUNT_DIR
+
+# Run integration tests for read_large_files with --only-dir mounting.
```

Comment:
you are not running with --only-dir as I only see static mounting in read_large_files_test.go?

Reply:
It is increasing time. One test took around 2 mins to run, that's why I keep only with static mounting.